src/components/Home/WalletCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { Card } from '@/components/0_Bruddle'
+import Image from 'next/image'
+import { motion } from 'framer-motion'
+import classNames from 'classnames'
+import Link from 'next/link'
+import { formatUnits } from 'viem'
+import PeanutWalletIcon from '@/assets/icons/peanut-wallet.png'
+import Icon from '@/components/Global/Icon'
+import { shortenAddressLong } from '@/utils'
+import { IWallet } from '@/interfaces'
+
+interface WalletCardProps {
+    type: 'wallet' | 'add'
```

Comment:
Thought:

Shouldn't we have 3 types?

- peanut smart wallet
- externally connected wallet
- add

And potentially in the future:
- watch wallets and non evm wallets

Reply:
Absolutely correct, right now we don't have much done in terms of external wallet, when I extracted this component we only have placeholders